tensorflow/python/kernel_tests/cwise_ops_test.py

```diff
@@ -1772,6 +1772,14 @@ def testSqrt(self):
 
 class RoundingTest(tf.test.TestCase):
 
+  def _compare_values(self, x, y=None, use_gpu=True):
```

Comment:
Yeah, it's controlled with a higher level run.  Our CI will test on GPU and CPU  (Hence the CPU tests and the GPU test being separate CI instances).